src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2693,13 +2704,38 @@ signalapi.readyForSignal("soft_stop")
 //Display current job signals
 println("Job signals "+ signalapi.getJobSignals())
 
-//Wait util the 'soft_stop' signal is sent (via the Workflow Execution portal or REST API)
+//Wait util the 'soft_stop' signal is sent (via the Workflow Execution portal, Scheduler portal or REST API)
 signalapi.waitFor("soft_stop")
 
 //Remove the 'soft_stop' signal from the set of job signals
 signalapi.removeSignal("soft_stop")
 ----
 
+Here is another example of a groovy code that uses some Signal API operations with signal variables.
+
+----
+import org.ow2.proactive.scheduler.common.job.JobVariable
+
+//Declare the signal variables
+List <JobVariable> signalVariables = new java.util.ArrayList<JobVariable>()
+signalVariables.add(variables.get("INTEGER_VARIABLE"))
+signalVariables.add(variables.get("BOOLEAN_VARIABLE"))
+
+//Declare that the current job is ready to receive and handle the 'soft_stop' signal with the given variables
+signalapi.readyForSignal("soft_stop", signalVariables)
+
+//Display current job signals
+println("Job signals "+ signalapi.getJobSignals())
+
+//Wait util the 'soft_stop' signal with variables is sent (via the Workflow Execution portal, Scheduler portal or REST API)
+outputParameters = signalapi.waitFor("soft_stop")
+
```

Comment:
please add a comment before this line explaining what the structure of outputParameters is (I guess this is a hashmap containing the names of the variables and their values entered by the user)